bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -3527,7 +3552,23 @@
         <comment>New roles and privileges for bahmni</comment>
         <sqlFile path="V1_96_NewRolesAndPrivileges.sql"/>
     </changeSet>
-    <changeSet id="map-users-to-new-roles" author="Jaswanth, Ravindra">
+    <changeSet id="bahmni-core-201906201151" author="Samuel34">
+    	<preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                select count(*) from users where system_id='superman';
+            </sqlCheck>
+        </preConditions>
+        <comment>Adding superman user</comment>
+        <sql>
+            SET @puuid = uuid();
+            INSERT INTO person(gender, birthdate_estimated, dead, creator, date_created, uuid) VALUES('M', 0, 0, 1, now(), @puuid);
+            SELECT person_id INTO @person_id from person where uuid = @puuid;
+            INSERT INTO person_name(person_id, preferred, given_name, family_name, creator, date_created, uuid) VALUES(@person_id, 1, 'Super', 'Man', 1, now(), @puuid);
+            INSERT INTO users(system_id, creator, date_created, person_id, uuid, username, password, salt) VALUES ('superman', 1, now(),@person_id, uuid(), 'superman', 'e28e3ae84c66bfba6b2c50a406567f3e34fea1d76b17c006931571fe5d940f6c6b81e49cf8ea5e0adfca19fe3beb68d8ad79f7c3812e92b8d502a9afcf2029b2', '1c9d7e94aeeb7a2459ef45ed200b2944582e0e7088d75f9b57a3644861ea766c20a269b3fe2eadaff1bc445ecfbd9bd3c0c550dfd813de48d39423cd3d1a8b10');
+            INSERT INTO provider (person_id, identifier, creator, date_created, uuid, name) VALUES (@person_id, 'superman', 1, now(), uuid(), 'superman');
+        </sql>
+    </changeSet>
+    <changeSet id="bahmni-core-201906170420" author="Jaswanth, Ravindra, Samuel34">
```

Comment:
why would we check against the bahmni-user role in the precondition? I dont see any query subsequently to do anything with the "bahmni-user" role

Reply:
Well... I see not much reason to have that precondition. Don't even know what that 'Bahmni-User' is really. Do you want me to remove this?